SceneLoader/SceneResourceSet.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+namespace SceneLoader
```

Comment:
All source should have the licensing headers added

Reply:
There are 2 different headers being used in this repo. Is that on purpose?

---
In reply to: [280248474](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280248474) [](ancestors = 280248474)